wamr-sdk/README.md

```diff
@@ -1,133 +1,3 @@
-
-
-
-### Menu configuration for building SDK
-
-Menu configuration is supported for easy integration of runtime components and application libraries for  the target architecture and platform. Run following command to start the menu config.
-
-```
-cd wamr-sdk
-./build_sdk.sh -i -n [profile name]
-```
-
-  The argument "-i" will make the command enter menu config mode as depicted below.
-
-<img src="../doc/pics/wamr_menu_config.png" alt="wamr build menu configuration" style="zoom:80%;" />
-
-After the menu configuration is finished, the profile config file is saved and the building process is automatically started. When the building gets successful, the SDK package is generated under folder $wamr-sdk/out/{profile}, and the header files of configured components were copied into the SDK package.
-
-
-
-### Build WASM applications with APP-SDK
-
-The folder “**app-sdk**” under the profile output directory contains all the header files and WASM library for developing the WASM application. For C/C++ based WASM applications, the developers can use conventional cross-compilation procedure to build the WASM application.  According to the profile selection of libc, following cmake toolchain files under folder [wamr-sdk/app](./app) are available for cross compiling WASM application:
-
-- ` wamr_toolchain.cmake` 
-- `wasi_toolchain.cmake`
-
-
-
-Refer to [build WASM applications](../doc/build_wasm_app.md) for the details.
-
-
-
-### Use Runtime SDK to build native application
-
-The output folder "**runtime-sdk**" contains all the header files and library files for integration with project native code. 
-
-You can link the pre-built library:
-``` cmake
-link_directories(${SDK_DIR}/runtime-sdk/lib)
-include_directories(${SDK_DIR}/runtime-sdk/include)
-# ......
-target_link_libraries (your_target vmlib -lm -ldl -lpthread)
-```
-
-This method can also be used when you don't use cmake
-
-You can refer to this sample: [CMakeLists.txt](../samples/simple/CMakeLists.txt).
-
-> NOTE: If you are familiar with how to configure WAMR by cmake and don't want to build the SDK, you can set the related settings on the top of your `CMakeLists.txt`, then the `runtime_lib.cmake` will not load settings from the SDK.
-
-
-
-### Integrate WAMR without pre-built WAMR library
-
-Use the provided `runtime_lib.cmake` file:
-
-You can include `${WAMR_ROOT}/cmake/runtime_lib.cmake` in your project's `CMakeLists.txt` file:
-
-``` cmake
-include (${WAMR_ROOT}/cmake/runtime_lib.cmake)
-add_library (vmlib ${WAMR_RUNTIME_LIB_SOURCE})
-# ......
-target_link_libraries (your_target vmlib -lm -ldl -lpthread)
-```
-
-You can refer to to product-mini building for Linux: [`product-mini/platforms/linux/CMakeLists.txt`](../product-mini/platforms/linux/CMakeLists.txt).
-
-> 
+# Notice
+`wamr-sdk` has been migrated to **[wamr-app-framework/wamr-sdk](https://github.com/bytecodealliance/wamr-app-framework/tree/main/wamr-sdk)**.
+Only some necessary files that A will use are retained here
```

Comment:
Seems all wamr-sdk files are migrated? Had better remove this line?

Reply:
We have retained some files that wasm app will use, like `libc-builtin-sysroot`. So I think we need to keep this line.